src/Orc.Controls/Helpers/Postpone/PostponeAction.cs

```diff
@@ -0,0 +1,76 @@
+﻿namespace Orc.Controls;
+
+using System;
+
+public sealed class PostponeAction
+{
+    private readonly Action _action;
+    private readonly IPostponeActionTimer _timer;
+
+    private bool _disposed;
+
+    public PostponeAction(Action action, IPostponeActionTimer timer)
+    {
+        ArgumentNullException.ThrowIfNull(action);
+        ArgumentNullException.ThrowIfNull(timer);
+
+        _action = action;
+        _timer = timer;
+        _timer.Tick += OnTimerTick;
+    }
+
+    public PostponeAction(Action action)
+        : this(action, new DispatcherTimerWrapper())
+    {
+    }
+
+    public static void Execute(Action action, int delay)
```

Comment:
This will make unit testing hard(er)

Reply:
but usage became better - i think in this case it would be enough  to text Execute (non static)